src/common/math/Constant.h

```diff
@@ -0,0 +1,61 @@
+// Generated with tools/math-constant/math-constant.py
+// Do not modify!
+
+#ifndef COMMON_CONSTANT_H_
+#define COMMON_CONSTANT_H_
+
+namespace Math {
+	//        Type  Name            Value           Operation  GNU          std::numbers
+	constexpr float phi_f         = 1.61803399f; // φ                       phif
+	constexpr float egamma_f      = 1.56746826f; // Γe                      egammaf
+	constexpr float ln2_f         = .693147181f; // ln(2)      M_LN2f       ln2f
+	constexpr float ln10_f        = 2.30258509f; // ln(10)     M_LN10f      ln10f
+	constexpr float log2e_f       = 1.44269504f; // log2(e)    M_LOG2Ef     log2ef
+	constexpr float log10e_f      = .434294482f; // log10(e)   M_LOG10Ef    log10ef
+	constexpr float pi_f          = 3.14159265f; // π          M_PIf        pif
+	constexpr float mul2_pi_f     = 6.28318531f; // 2π
+	constexpr float inv_pi_f      = .318309886f; // 1÷π        M_1_PIf      inv_pif
+	constexpr float inv_mul2_pi_f = .159154943f; // 1÷2π
+	constexpr float div2_pi_f     = .636619772f; // 2÷π        M_2_PIf
+	constexpr float div180_pi_f   = 57.2957795f; // 180÷π
+	constexpr float divpi_2_f     = 1.57079633f; // π÷2        M_PI_2f
+	constexpr float divpi_4_f     = .785398163f; // π÷4        M_PI_4f
+	constexpr float divpi_90_f    = .034906585f; // π÷90
+	constexpr float divpi_180_f   = .017453292f; // π÷180
+	constexpr float divpi_360_f   = .008726646f; // π÷360
```

Comment:
The ones less than 0.1 seem to have less precision. You should do like `%.9g` not `%.9f`

Reply:
It's me who actually stripped that to have nice aligned columns! 🤦‍♀️️ I forgot I could aligne the numbers to the right. Now fixed.